modules/ROOT/examples/css-codeblock/footnotes-plugin.css

```diff
@@ -0,0 +1,15 @@
+.mce-footnote {
+
+}
+
+.mce-footnotes {
+
+}
+
+.mce-footnotes-backlink {
+
+}
+
+.mce-footnotes-note {
+
+}
```

Comment:
This does not match what we provide... it's meant to contain the CSS we include so that customers can take that and alter it.

- [ ] Fix missing CSS

Reply:
Missing CSS has been added

Reply:
It hasn't been added sorry by the looks, only parts of it has.

Reply:
I initially thought we wouldn't need/want to include the print-specific CSS as it is more applicable to printing within the editor. Although I didn't really think that through I realised as an integrator might want to override CSS and like to know all of the defaults. I'll add it back.

I assume that is the part you are saying is missing?

Reply:
Yeah, that's correct. In addition someone may still want to print it outside the editor and have it behave the same.

Reply:
Cool. Thanks for picking up

Reply:
New commit has been pushed